providers/aws/fargate/pod.go

```diff
@@ -51,6 +54,7 @@ type Pod struct {
 	taskCPU         int64
 	taskMemory      int64
 	containers      map[string]*container
+	taskRoleArn     *string
```

Comment:
Can you please move this to between taskArn and taskStatus? So that all ARNs are together. Also, any reason for the type to be *string? Let's just store it as string, like the others.